### PR TITLE
Code cleanup (removed unused variables and imports)

### PR DESCRIPTION
**Pull request by andrewhong04**
I got all my information from: https://lgtm.com/projects/g/Tuxemon/Tuxemon/
This could boost Tuxemon's grade from a B to an A?

### CODE REVIEW

File: scripts/TestClient.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 from tuxemon.mod_manager import Manager
 import os
-import readline
```
Comment (magmaus3) [inline scripts/TestClient.py]: Note that `readline` is imported for the input history (accessible by the up/down arrows)

File: tuxemon/event/actions/random_encounter.py [hunk 2]
```diff
@@ -124,7 +124,6 @@ def update(self) -> None:
             self.stop()
 
     def cleanup(self) -> None:
-        npc = None
```
Comment (freshreplicant) [inline tuxemon/event/actions/random_encounter.py]: This code is changed in #1117, see https://github.com/Tuxemon/Tuxemon/pull/1117/commits/5de6f9e532ff2b8a6d225876402f8281e76d5b2f.

This PR is needed as it stops the game from crashing on random encounters.

**Comment by freshreplicant** [comment]
I'm afraid I can't really review this code, I'd be in way over my head.

Did you make a local branch with these changes to test if they break anything?

**Comment by andrewhong04** [comment]
> I'm afraid I can't really review this code, I'd be in way over my head.
> 
> Did you make a local branch with these changes to test if they break anything?

Yup, this is on a local branch. I'll play the game for 20 minutes later to see if I encounter any errors.

**Comment by xirsoi** [comment]
While this *looks* fine to me, I wanna make sure it's thoroughly tested. Wild battles, trainer battles, buying and selling, everything needs to be tested.

**Review by bitcraft — APPROVED** [review]

**Pull request merged**